Source/Core/Core/IOS/ES/ES.cpp

```diff
@@ -271,9 +276,9 @@ bool ES::LaunchIOS(u64 ios_title_id)
     if (!tmd.IsValid() || !ticket.IsValid() || !tmd.GetContent(tmd.GetBootIndex(), &content) ||
         !m_ios.BootIOS(ios_title_id, GetContentPath(ios_title_id, content)))
     {
-      PanicAlertT("Could not launch IOS %016" PRIx64 " because it is missing from the NAND.\n"
```

Comment:
Is the `#include <cinttypes>` still necessary after the removal of those macro uses?